teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission, name)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
```

Comment:
- [ ] Please do the same with this one, correct the `super` method and use `require_relative`! 🤓 🔧